pkg/porter/dependencies.go

```diff
@@ -181,6 +181,11 @@ func (e *dependencyExecutioner) prepareDependency(dep *queuedDependency) error {
 	dep.CNABFile = cachedDep.BundlePath
 	dep.RelocationMapping = cachedDep.RelocationFilePath
 
+	invocationImage := cachedDep.Bundle.InvocationImages[0]
```

Comment:
Do you think we could check the digest when we pull the bundle, instead of later when we try to run it? That way the logic would be in a single spot instead of split across dependencies and the other code paths?

Reply:
Good call, thank you.  Updated.